cli/popper/commands/cmd_view.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+import click
+import hcl
+import popper.utils as pu
+from ..cli import pass_context
+import types
+
+
+@click.argument(
+    'wfile', required=True)
+@click.command('view', short_help='Shows an ascii graph of a workflow')
+@pass_context
+def cli(ctx, wfile):
+    """Displays an ascii graph of a workflow"""
+    if not os.path.isfile(wfile):
+        pu.fail("File {} not found.\n".format(wfile))
+        exit(1)
+
+    with open(wfile, 'r') as fp:
+        wf = hcl.load(fp)
+
+    graph = "digraph G{\n"
+    name = list(wf["workflow"].keys())[0]
+    pu.info("Ascii for %s workflow\n" % name)
+
+    action = wf["workflow"][name]["resolves"]
+    parent_action = cur_action = action
+
+    if not isinstance(cur_action, str):
+        cur_action = cur_action[0]
+
+    graph = add(parent_action, cur_action, wf["action"], graph) + "}"
+
+    with open('ascii.dot', 'w') as fp:
+        fp.write(graph)
+
+    os.system("awk '!seen[$0]++' ascii.dot > out")
+    os.system('cat out | graph-easy --from=dot --as_ascii')
```

Comment:
hey @akashrajr1 , thank you so much for working on this! The goal is to have popper rely only on python modules. So what we can do is to leave this out and put it in the documentation. If we do this, then we maybe should rename this command to `popper dot` and print the `.dot` graph to `stdout`.

Another alternative would be to find a Python library that prints to ascii a `.dot` graph. Or maybe we can do both (a `popper dot` and a python-based `popper view`)?

Reply:
I actually, went by your original suggestion and tried to use this [python module](https://github.com/junjihashimoto/py-stacked-dag), but I was having problems running it. I found that `__init__.py` was corrupt for one of the directories, and I had to recreate it to make it work. So it brings in the question of usability. (I will have one more look, in case I missed something)
So I was wondering if we could use perl libraries instead , but no worries. 

Doing both commands sounds good to me. But for now, we can just print the `.dot` file to `stdout` using `popper dot` for now, until a solution presents itself for `popper view`. We can add a recommendation in the documentation to use 
```
apt install cpanminus
cpanm Graph::Easy
```

Reply:
@ivotron maybe when L39 fails for `graph-easy` we can prompt instruction to install it?

Reply:
The problem with instructions is that we don't know what the user is using. Are users on a mac, ubuntu, centos, arch, etc?

Reply:
@ivotron please review. Updated as requested